server/app/services/export/JsonExporter.java

```diff
@@ -25,32 +30,34 @@
 import services.applicant.ApplicantService;
 import services.applicant.JsonPathProvider;
 import services.applicant.ReadOnlyApplicantProgramService;
-import services.applicant.question.CurrencyQuestion;
-import services.applicant.question.DateQuestion;
-import services.applicant.question.MultiSelectQuestion;
-import services.applicant.question.NumberQuestion;
-import services.applicant.question.PhoneQuestion;
 import services.program.ProgramDefinition;
 import services.program.ProgramNotFoundException;
 import services.program.ProgramService;
-import services.question.LocalizedQuestionOption;
+import services.question.types.QuestionType;
 
 /** Exports all applications for a given program as JSON. */
 public final class JsonExporter {
 
   private final ApplicantService applicantService;
   private final ProgramService programService;
   private final DateConverter dateConverter;
-  private static final PhoneNumberUtil PHONE_NUMBER_UTIL = PhoneNumberUtil.getInstance();
+  private final QuestionJsonPresenter.Factory presenterFactory;
+
+  // Question types for which we should call getApplicationPath() on their path
+  // when constructing the API response.
+  private static final ImmutableSet<QuestionType> USE_APPLICATION_PATH_TYPES =
+      ImmutableSet.of(NAME, ID, TEXT, EMAIL, ADDRESS, DROPDOWN, RADIO_BUTTON, FILEUPLOAD);
```

Comment:
`USE_APPLICATION_PATH_TYPES` is based on the paths where we used to call `asApplicationPath()` on the switch statement:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/export/JsonExporter.java#L121-L219